src/components/core/ApplicationShell.tsx

```diff
@@ -17,10 +17,10 @@ export default function ApplicationShell({ children }: { children: React.ReactNo
 			}}
 			navbarOffsetBreakpoint="sm"
 			asideOffsetBreakpoint="sm"
-			navbar={<ApplicationNavigationBar opened={opened} />}
+			navbar={<ApplicationNavigationBar opened={opened} onClose={() => {true} } />}
```

Comment:
why? 🤔

Reply:
and that fixed it??? XD

Reply:
@Lxkas how do i fix it.

Reply:
in this function called, can we do something like 

`onClose{() => { setOpened(false) })`


so like this
```tsx
navbar={<ApplicationNavigationBar opened={opened} onClose={() => {setOpened(false)} } />}
```

Reply:
nvm, one edge case:

this will always close the navbar - won't be able to select the subitems if there's any